### PR TITLE
Bump sbt versions

### PR DESCRIPTION
**Pull request by eatkins**

### CODE REVIEW

**Review by eed3si9n — DISMISSED** [review]

File: build.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ import scala.util.Try
 
 ThisBuild / version := {
   // update .travis.yml too for dog fooding
-  val v = "1.4.2-SNAPSHOT"
+  val v = "1.4.3-SNAPSHOT"
```
Comment (eed3si9n) [inline build.sbt:12]: We need to bump `.travis.yml` as I wrote in the comment.

Comment (eatkins) [reply build.sbt:12]: Strangely that doesn't fix it: https://travis-ci.com/github/sbt/sbt/jobs/425031122

Comment (eed3si9n) [reply build.sbt:12]:
```bash
rm -r $(find $HOME/.sbt/boot -name "*-SNAPSHOT") || true
rm: missing operand
Try 'rm --help' for more information.
```

Comment (eed3si9n) [reply build.sbt:12]: Actually not sure if that would cause this.. I'm going to restart the job.

File: .travis.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ matrix:
   include:
     - env:
         - SBT_LOCAL=true
-        - SBT_VERSION_PROP=-Dsbt.version=1.4.2-SNAPSHOT
+        - SBT_VERSION_PROP=-Dsbt.version=1.4.3-SNAPHOT
```
Comment (eed3si9n) [inline .travis.yml:26]: SNAP HOT! - https://github.com/sbt/sbt/pull/6063

**Pull request closed without merging**